app/src/test/java/com/example/habitsmasher/ui/dashboard/HabitListTest.java

```diff
@@ -30,21 +30,13 @@ public void addHabit_validHabitAddition_expectHabitAddedToList(){
     }
 
     @Test
-    public void editHabit_validEdit_expectHabitToBeEdited() {
-        long habitId = 0;
-        Habit habit = new Habit("Title 1", "Reason 1", new Date(), habitId);
-        _habitList.addHabitLocal(habit);
-        Date newDate = new Date();
-        int habitToEdit = 0;
-        String newTitle = "Title 2";
-        String newReason = "Reason 2";
-        _habitList.editHabitLocal(newTitle, newReason, newDate, habitToEdit);
-        Habit editedHabit = _habitList.getHabitList().get(habitToEdit);
-
-        assertEquals(newTitle, editedHabit.getTitle());
-        assertEquals(newReason, editedHabit.getReason());
-        assertEquals(newDate, editedHabit.getDate());
-        assertEquals(habitId, editedHabit.getHabitId());
```

Comment:
Since the method was deleted, this test is no longer needed

Reply:
It looks like we'll still be needing this function then, so probably don't delete this test